packages/dapp-svelte-wallet/react-ui/src/index.js

```diff
@@ -4,15 +4,47 @@ import '@agoric/eventual-send/shim';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { CssBaseline, createTheme, ThemeProvider } from '@material-ui/core';
 import App from './App.js';
 import ApplicationContextProvider from './contexts/Application';
 
 Error.stackTraceLimit = Infinity;
 
+const appTheme = createTheme({
```

Comment:
It was good to lift this styling to the top level.  Maybe this is where the global CSS variables belong later?

Reply:
Yea that makes sense, or in an index.css